src/framework/draw/types/font.cpp

```diff
@@ -183,7 +183,7 @@ QFont Font::toQFont() const
 
 Font Font::fromQFont(const QFont& qf, Font::Type type)
 {
-    Font f(qf.family(), type);
+    Font f(qf.family().toStdString(), type);
```

Comment:
This converts back and forth between UTF-16 and UTF-8. Instead, use `Font(FontDataKey(qf.family()), …)`, or `Font(String::fromQString(qf.family()), …)` (or `Font(FontDataKey(String::fromQString(qf.family())), … )`)